content/en/announcements/devex-survey.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Developer Experience Survey
+date: 2024-12-16
+expiryDate: 2025-01-31
+---
+
+<i class="fas fa-bullhorn"></i> It takes just a few minutes to help us
+[improve the developer experience](https://docs.google.com/forms/d/1orPz5ayzosFrgYRm3-y90UMrt2ZjvIBKMDL_a2E3Fq8/viewform?ts=67575cca&edit_requested=true)!
+Survey is open through January 31, 2025.
```

Comment:
Here's a suggestion to shorten the banner. I think it's useful to mention the "5-10 min" estimate:

```suggestion
<i class="fas fa-bullhorn"></i> Help us [improve the developer experience] (5-10
min). Survey is closes January 31.

[improve the developer experience]:
  https://docs.google.com/forms/d/1orPz5ayzosFrgYRm3-y90UMrt2ZjvIBKMDL_a2E3Fq8/viewform?ts=67575cca&edit_requested=true
```